### PR TITLE
TTK-13983 service: Now always returns multiple tracks when available

### PR DESCRIPTION
**Pull request by Alfro**

### CODE REVIEW

File: Services/PaellaDataService.php [hunk 1]
```diff
@@ -150,44 +150,37 @@ private function getMmobjTracks(MultimediaObject $mmobj, $trackId)
...
+        }
+        foreach($presentationTracks as $track) {
+            if(in_array($track->getVcodec(), $availableCodecs)) {
+                $tracks['presentation'] = $track;
+                break;
             }
-            if($track)
-                $tracks['display'] = $track;
+        }
+        if($sbsTrack && $sbsTrack->getVcodec() == 'h264')
```
Comment (rubenrua) [inline Services/PaellaDataService.php]: Why not also use `$availableCodecs`

**Comment by rubenrua** [comment]
When the multimedia object is monostream, the track does not contain the `presenter/delivery` tag.

We need unit test for this services.

File: Services/PaellaDataService.php (hunk 1 above)
Comment (Alfro) [inline Services/PaellaDataService.php]: I saw it but somehow didn't change it.

**Comment by rubenrua** [comment]
Thank you.

+1 for the tests.

**Pull request merged**